### PR TITLE
Feature/jest tests

### PR DESCRIPTION
**Pull request by vincentaudebert**
@thibaudcolas to review this please. Merge if you are happy.

### CODE REVIEW

**Comment by thibaudcolas** [comment]
Also please update the relevant documentation:

https://github.com/springload/frontend-starter-kit/blob/master/docs/frontend-launch-checklist.md#automated-tests, https://github.com/springload/frontend-starter-kit#tests, https://github.com/springload/frontend-starter-kit/tree/master/docs#tooling

File: package.json [hunk 1]
```diff
@@ -21,9 +21,7 @@
     "lint:sass": "npm run linter:sass -s",
     "report:css": "gulp report:css",
     "lint": "npm run lint:versions -s && npm run lint:js -s && npm run lint:sass -s",
-    "test": "npm run test:unit -s",
-    "test:unit": "mocha --compilers js:babel-register core/static_src/js/tests",
```
Comment (vincentaudebert) [inline package.json]: Done. Will repop shrinkwrap also.

File: core/static_src/js/tests/analytics.spec.js [hunk 2]
```diff
@@ -1,80 +1,80 @@
-import { expect } from 'chai';
-import sinon from 'sinon';
 import { initErrorTracking, social, errors } from '../api/analytics';
```
Comment (vincentaudebert) [inline core/static_src/js/tests/analytics.spec.js]: Yeah it seems it works without it.

File: core/static_src/js/tests/format.spec.js [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
 
         it('number formatting makes the numbers look nicer', () => {
-            expect(humanNumber(0)).to.equal('0');
-            expect(humanNumber(5000)).to.equal('5,000');
-            expect(humanNumber(5000000)).to.equal('5,000,000');
-            expect(humanNumber(-5000)).to.equal('-5,000');
-            expect(humanNumber(-5000000)).to.equal('-5,000,000');
-            expect(humanNumber(5000.5000)).to.equal('5,000.5');
-            expect(humanNumber(0.3333333)).to.equal('0.3333333');
+            expect(humanNumber(0)).toBe('0');
```
Comment (vincentaudebert) [inline core/static_src/js/tests/format.spec.js]: https://facebook.github.io/jest/docs/api.html#toequalvalue

Yeah I was about to too and then I checked the doc :)
toEqual is to test 2 objects, not values like strings or number. toBe is the one for "simple" variables.

**Comment by thibaudcolas** [comment]
Merged

**Pull request merged**